### ISSUE TITLE
Bake inn generasjontid og vurderingsperiode i eksisterende tekst

### ISSUE THREAD

**Issue by SiriKasperdatter** [Artsdatabanken/assessments-frontend#401]
![image](https://user-images.githubusercontent.com/61016662/141964109-bdb90b59-0b24-429f-9c1b-01246d787e07.png)

Kunne vi endret der generasjonstiden og vurderingsperioden til inn i setningen over? Må naturligvis gjelde for både A- og C-kriteriet.

Ser for meg følgende tekst (ikke 100 % fornøyd med forslaget, så veldig åpen for forslag): 

Reduksjon siste 3 generasjoner (generasjonstid: 33 år) som gir en vurderingsperiode på 99 år. Reduksjonen eller årsakene til reduksjonen trenger ikke å ha opphørt, være forstått eller reversible.

**Comment by helemork** [comment]
Her kan vi:

@SiriKasperdatter  her trenger jeg litt mer fedback for å gjøre noe.

**Enkel løsning:**
Sette inn generasjonstid via en tag rett i jsonen som byttes ut når vi rendres, eller splitte jsonen i to og rendre setningen i midten. Førsntevnte er minst å endre på. 

Eksempel:
Reduksjon siste 3 generasjoner **xxx**, men minimum 10 år **zzz**. Reduksjonen eller årsakene til reduksjonen trenger ikke å ha opphørt, være forstått eller reversible.
  - xxx byttes ut med din ønskede string: "(generasjonstid: 33 år)"
  - zzz byttes ut med din ønskede string: "som gir en vurdringsperiode på 99 år"  

Fordel: enkel, lite tenking, rett frem
Ulempe: Tar ikke høyde for f.eks mer eller mindre enn minimum og maksimum.

**Litt mer kompleks løsning:** 
Som over, men også ha ",men minimum 10 år" og evt. andre ting som noe som slenges inn etterpå. Jeg synes forøvrig og vi skulle hatt med max, siden vi faktisk har en max. Greit nok at de ikke trigges, men de er der, og det er trær og hvaler som kan bli over 100år.

**Comment by SiriKasperdatter** [comment]
Liker forslag til løsning! Men det går ikke å legge inn min og maks altså? Det er mange arter med generasjonstid 1 som da har vurderingsperiode på 10 år, og som du sier en del som har lengre generasjonstid enn 33 år.

**Comment by helemork** [comment]
Jo, det meste går an. Jeg mente mer jeg synes at så lenge kravet alltid står, er det rart å ikke alltid vise det? Tenker vi kan jo dumpe tekstringen i glossary kanskje.

Er min og max lik for A og C? Jeg husker ikke helt i farta

**Comment by SiriKasperdatter** [comment]
På A-kriteriet står det innledningsvis, så da trenger det ikke stå en gang til.

Men på C-kriteriet presiserer vi det ikke, nettopp fordi der er min og maks ulik, både sammenlignet med A-kriteriet og innad for de ulike kriteriene. 
<img width="265" alt="image" src="https://user-images.githubusercontent.com/61016662/145547760-e02960fb-67fe-4425-bfa7-7638c42ae654.png">
